native/src/components/NewsDetail.tsx

```diff
@@ -56,6 +56,11 @@ const NewsDetail = ({ theme, newsItem, language, navigateToLink }: PropsType): R
     },
     [navigateToLink, language]
   )
+
+  const decodedNewsItemTitle = parseHTML(newsItem.title, {
```

Comment:
Why do we need this here, you decoded it in the api-client already, didn't you? At least for Tünews. I think we should probably completely decode in the api-client or completely decode in web and native.